src/storage/mod.rs

```diff
@@ -192,6 +192,11 @@ where
     T: Component,
     D: Deref<Target = MaskedStorage<T>>,
 {
+    /// Gets the wrapped storage.
+    pub fn storage(&self) -> &T::Storage {
```

Comment:
Could we name this `unprotected_storage`? I usually prefer short names, but in this case it's not what most people are looking to use, so I think it's good to have some kind of warning.